### PR TITLE
[FIX] sms: wrong message error

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**
* When checking model coherency for sms, if choose transient model -> raise ValidationError message but the message seem not follow with the code so much

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by duong77476-viindoo** [comment]
Hello @tde-banana-odoo Can you check please ?

**Comment by duong77476-viindoo** [comment]
Hello @tde-banana-odoo Wrong typo i suppose

File: addons/sms/models/ir_actions_server.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def _check_model_coherency(self):
         super()._check_model_coherency()
         for action in self:
             if action.state == 'sms' and (action.model_id.transient or not action.model_id.is_mail_thread):
-                raise ValidationError(_("Sending SMS can only be done on a mail.thread or a transient model"))
+                raise ValidationError(_("Sending SMS can only be done on a mail.thread or not a transient model"))
```
Comment (tde-banana-odoo) [inline addons/sms/models/ir_actions_server.py:52]: Indeed ... but it is an and between the two, hence
```suggestion
                raise ValidationError(_("Sending SMS can only be done on a not transient mail.thread"))
```

Comment (duong77476-viindoo) [reply addons/sms/models/ir_actions_server.py]: Sure, as you wish sir <3

File: addons/sms/models/ir_actions_server.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def _check_model_coherency(self):
         super()._check_model_coherency()
         for action in self:
             if action.state == 'sms' and (action.model_id.transient or not action.model_id.is_mail_thread):
-                raise ValidationError(_("Sending SMS can only be done on a mail.thread or a transient model"))
+                raise ValidationError(_("Sending SMS can only be done on a not transient mail.thread"))
```
[the remark this answers is not in the document]
Comment (duong77476-viindoo) [reply addons/sms/models/ir_actions_server.py:52]: Ah ha, done

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**